schwifty/checksum/netherlands.py

```diff
@@ -0,0 +1,24 @@
+import functools
+
+from schwifty import checksum
+from schwifty import registry
+
+
+register = functools.partial(checksum.register, prefix="NL")
+
+
+@register
+class DefaultAlgorithm(checksum.Algorithm):
+    name = "default"
+    accepts = checksum.InputType.BBAN
+
+    def compute(self, bban: str) -> str:
+        raise Exception('Cannot compute checksum for Netherlands')
+
+    def validate(self, bban: str) -> bool:
+        acc_number = bban[-10:]
+        total = 0
+        for i, digit in enumerate(map(int, acc_number)):
+            total += i * (10-i)
```

Comment:
Are you certain that this is implementation is correct? It only uses the index variable `i` and never makes use of the `digit` at the corresponding position. Do you maybe have some reference documentation for the algorithm itself?

Reply:
damn, yeah sorry, my mistake.. should be `total += digit * (10-i)`

I wrote this a while ago, don't remember the source though, I remember it took me a long time googling
```
# Netherlands account number algorithm uses this factor table:
# digit | factor
# 1st   | 10
# 2nd   | 9
# ...
# 10th  | 1
# 
# After multiplying digits according to the table, sum of these results must be divisible with 11
```